.github/workflows/test_build_wheels_windows_arm64.yml

```diff
@@ -0,0 +1,45 @@
+name: Test Build Windows Wheels ARM64
+
+on:
+  pull_request:
+    paths:
+      - .github/workflows/test_build_wheels_windows_arm64.yml
+      - .github/workflows/build_wheels_windows.yml
+      - .github/workflows/generate_binary_build_matrix.yml
+      - tools/scripts/generate_binary_build_matrix.py
+  workflow_dispatch:
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  generate-matrix:
+    uses: ./.github/workflows/generate_binary_build_matrix.yml
+    with:
+      package-type: wheel
+      os: windows
+      test-infra-repository: ${{ github.repository }}
+      test-infra-ref: ${{ github.ref }}
+      with-cuda: disable
+      architecture: arm64
+  test:
+    needs: generate-matrix
+    strategy:
+      fail-fast: false
+      matrix:
```

Comment:
Do we also want to test audio here as well?

Reply:
Does it matter which one we use in the testing workflow? I thought this worklflow is supposed to just to check if anything starts breaking, so does it make a difference if we use audio or vision?

Reply:
Well if you want to test if this works across vision and audio it might make sense to test both of them here